scripts/core/activity/views/activity-modal.html

```diff
@@ -1,4 +1,5 @@
-<div style="z-index: 1051 !important;" class="modal in {{activity.cssClass}}" ng-if="activity">
+<div  ng-style="!activity.zIndex ? {'z-index': '1051'} : {'z-index': activity.zIndex}"  class="modal in {{activity.cssClass}}" ng-if="activity">
+    <div>ja sam taj 123</div>
```

Comment:
the fix is 1 line and you still manage not to see that you're comitting test code? :laughing:

Reply:
unbelievable... no comment